README.md

```diff
@@ -25,10 +26,16 @@ The following table provides a detailed list of all models materialized within t
 |--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [sap__0gl_account_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0gl_account_attr)    |  Access the general ledger account number attribute.              |
-| [sap__0material_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0material_attr)    |  Access the material master data from the source system.                                                                                                     |
+| [sap__0comp_code_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0comp_code_attr)    |  Access the company code attributes.                               |
+| [sap__0comp_code_text](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0comp_code_text)    |  Access the company code text information.   |
+| [sap__0company_text](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0company_text) |  Access the company text information.   |
+| [sap__0customer_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0customer_attr)    |  Access the customer master data attributes.                               |
 | [sap__0fi_gl_10](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0fi_gl_10)    | Access the totals records from the leading ledger in the new general ledger.  |
-| [sap__0fi_gl_14](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0fi_gl_14)  | Access line items of the leading ledger,  contains all line items that have been extracted from the source system.                                                                     |                                                                    
+| [sap__0fi_gl_14](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0fi_gl_14)  | Access line items of the leading ledger,  contains all line items that have been extracted from the source system.                                                                  |                            
+| [sap__0gl_account_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0gl_account_attr)    |  Access the general ledger account number attribute.              |
+| [sap__0material_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0material_attr)    |  Access the material master data from the source system.                                                          |
+| [sap__0vendor_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0vendor_attr)    |  Access the vendor master data attributes.                               |
+| [sap__0vendor_text](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0vendor_text) |  Access the vendor text information.   |
 
 # 🎯 How do I use the dbt package?
 ## Step 1: Prerequisites
```

Comment:
The prerequisites from the source package should also be included in this package. This way users are in the loop regarding what is needed to run this package without needing to also look at the source package.

Reply:
Prerequisite tables added

Reply:
Small request to edit the prerequisites section. I feel we should make it a bit more clear (especially around the range of SAP connectors that can be used. My thoughts are something like the following:

```md
## Step 1: Prerequisites
To use this dbt package, you must have the following:
- At least one Fivetran of the following SAP connectors:
   - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
   - [HVA SAP ECC](https://fivetran.com/docs/databases/sap-erp/high-volume-agent/hva-sap-ecc-hana)
   - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana) 
-  Within the connector, syncing the following respective tables into your destination:
  - bkpf
  - bseg
  - faglflexa
  - faglflext
  - kna1
  - lfa1
  - mara
  - pa0000
  - pa0001
  - pa0007
  - pa0008
  - pa0031
  - ska1
  - t001
  - t503
  - t880
- A **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, **Databricks** destination.

```